pl.js

```diff
@@ -0,0 +1,688 @@
+server.config.allowSpect Obserwowanie
+server.config.killRewards Nagrody za zabójstwa
+server.config.headshotOnly Tylko strzały w głowę
+server.config.noSecondary Wyłącz drugą broń
+server.config.noStreaks Wyłącz serie zabójstw
+server.config.canSlide Ślizganie
+server.config.wallJP Siła odbijania od ścian
+server.config.autoJump Automatyczne skakanie
+server.config.bDrop Upadek pocisków
+server.config.thirdPerson Tryb trzeciej osoby
+server.config.requireCaptcha Wymagaj captch'y
+server.config.nameTags Ukryj nazwy nad graczami
+server.config.noReload Bez przeładowywania
+server.config.private Prywatna gra
+server.config.krRewards Nagrody KR
+server.config.warmupTime Czas rozgrzewki
+server.config.objtvTime Czas miejsca celu
+server.config.forceSpawn Czas automatycznego odrodzenia
+server.config.scoreLimit Limit wyniku
+server.config.keepTScore Zatrzymaj wynik zespołu
+server.config.forceC Wymuś klasy trybu
+server.config.logTim Czas logów (Wyścig, Parkour)
+server.config.hitBoxPad Hitbox Padding
+server.config.hpRegen Regeneracja życia
+server.config.disableB Wyłącz obramowanie
+server.config.throwMel Rzucanie broni białych
+server.config.noCosm Wyłącz elementy kosmetyczne
+social.login Loguj/Rejestruj
+social.login.failed Logowanie nie powiodło się
+social.login.buy Zaloguj się by kupować
+social.login.sell Zaloguj się by sprzedawać
+social.login.gift Zaloguj się by darować
+social.login.proccessing Logowanie...
+social.since Od
+social.user.maps Gry użytkowników
+social.user.mods Modyfikacje użytkowników
+social.maps.none Nie znaleziono gier...
+social.mods.none Nie znaleziono modyfikacji...
+social.market.price.low Najniższa cena
+social.market.price.list Pokaż cenę w
+social.market.searching Wyszukiwanie
+social.market.listed wystawione {0} temu
+social.market.list Wystaw przedmiot
+social.market.unlist Zdejmij przedmiot
+social.market.unlisting Zdejmowanie przedmiotu...
+social.market.posting Postowanie przedmiotu...
+social.market.req poziom {0} wymagany
+social.market.cant Nie stać cię na to
+social.market.purchase Kup
+social.market.fee Brak podatku od wystawienia
+social.market.failed Niepowodzenie. Spróbuj ponownie
+social.market.head Przedmiot na głowę
+social.market.back Przedmiot na plecy
+social.market.waist Przedmiot na talię
+social.market.face Przedmiot na twarz
+social.market.shoe Przedmiot butów
+social.market.pet Przedmiot zwierzaka
+social.market.melee Przedmiot broni białej
+social.market.spray Przedmiot spreju
+social.market.dye Przedmiot barwnika
+social.market.none Nie znaleziono przedmiotów
+social.market.login Musisz być zalogowany.
+social.market.grace gotowy w {0}
+social.market.sort Filtruj/Sortuj
+social.market.filter Wyszukaj
+social.market.sort2 Sortuj
+social.market.price.high Najwyższa cena
+social.trading.submitting Wysyłanie ofery...
+social.market.gift.amount Wartość podarunku w
+social.market.fee.gift Brak podatku
+social.market.gift Wyślij KR
+social.market.gift.msg Zostaw wiadomość
```

Comment:
Here I am putting the translation (yes, this is my own translation of them) of lines 696-747 from the en.js file:
respawn.auto Automatyczne odrodzenie za {0}
custom.host.ranked Nie możesz hostować gier, gdy grasz w trybie rankowanym.
funds.insufficient Niewystarczające fundusze
funds.out Wygląda na to, że skończyło Ci się KR.
custom.passcode.long Zbyt długie hasło
custom.passcode.input Wpisz hasło serwera, aby dołączyć
custom.tryagain Spróbuj ponownie
purchase.kr.success Dodano {0} KR do Twojego konta
limited.purchase.ask Czy jesteś pewien, że chcesz kupić [{0}] za {1} KR?
limited.purchase.success Dodano {0} do Twojego ekwipunku
pretend.exit wciśnij [{0}] aby wyjść
settings.loaded Załadowano ustawienia
settings.loaded.none Nie znaleziono ustawień
account.login.required Wymagane jest bycie zalogowanym
account.banned.hack To konto zostało zbanowane za używanie cheatów
account.banned.appeal Jeśli chcesz odwołać się od bana, skontaktuj się z:
account.delete Możesz zgłosić żądanie usunięcia Twojego konta, jeśli nie cofniesz tego żądania w ciągu 14 dni od wysłania go, Twoje konto zostanie permanentnie usunięte.
account.delete.request Czy jesteś pewien, że chcesz zgłosić żądanie usunięcia Twojego konta?
account.delete.cancel Czy jesteś pewien, że chcesz cofnąć żądanie usunięcia Twojego konta?
spin.free.available DARMOWE KR DOSTĘPNE!
spin.free.claim Można zakręcić tym kołem raz na godzinę po rozegraniu co najmniej jednego meczu.
spin.free.disable Te wyskakujące okienka można wyłączyć w ustawieniach
mod.reset.0 Zmiana zacznie obowiązywać po odrodzeniu.
mod.reset.1 Obiekty na mapie nie ulegają resetowi
windows.profile.picture Zmień zdjęcie profilowe
stats.assists Asysty
stats.nukes Bomby atomowe
stats.melee Zabójstwa przy użyciu broni białej
stats.fistkills Zabójstwa w wyniku pobiciu
stats.thrownkills Zabójstwa w wyniku rzutu bronią białą
stats.headshots Trafienia w głowę
stats.wallbangs Trafienia przez ściany
stats.sprays Użycia sprejów
windows.host.default.settings Włączenie domyślnych ustwaień pozwala na przechowywanie postępu graczy oraz najwyższych wyników
windows.host.default.commands *Komendy hosta są wyłączone w tym trybie: /noclip /god /smite etc
windows.host.rewards Włączenie nagród KR zmusza serwer do używania domyślnych ustwaień i pozwala graczom zdobywać KR z funduszy mapy:
windows.store.wheels.sub Wypróbuj swoje swoje szczęście i zakręć kołem, aby móc wygrać nowe skórki!
windows.store.currency * Ceny w USD mogą być przeilczone na Twoją lokalną walutę po wartości wymiany ustalonej przez dostawcę płatności lub bank
windows.store.creator Pomyśl nad wsparciem swoich ulubionych twórców treści gry (CC) używając ich kodu
windows.chall.customs Nie można zdobywać wyzwań w niestandardowych meczach
twitch.link Czy jesteś pewien, że chcesz połączyć konto gry z kontem na Twitchu?
twitch.unlink Czy jesteś pewien, że chcesz rozłączyć konto gry z kontem na Twitchu?
windows.clan.leave Czy jesteś pewien, że chcesz opuścić ten klan?
windows.clan.delete Czy jesteś pewien, że chcesz usunąć ten klan?
windows.clan.kick Czy jesteś pewien, że chcesz wyrzucić {0} z klanu?
windows.store.premium.purchase Czy jesteś pewien, że chcesz kupić {0} dni Premium?
windows.playerlist.ban Czy jesteś pewien, że chcesz wyrzucić {0} z tej gry?
windows.playerlist.flag Czy jesteś pewien, że chcesz oflagować {0}?
windows.playerlist.unflag Czy jesteś pewien, że chcesz odflagować {0}?
windows.playerlist.punish Czy jesteś pewien, że chesz podjąć akcję dotyczącą {0}?
`;

Reply:
"*Komendy hosta są wyłączone w tym trybie: /noclip /god /smite etc"
Change "etc" to "itp"
"Włączenie nagród KR zmusza serwer do używania domyślnych ustwaień i pozwala graczom zdobywać KR z funduszy mapy:"
*ustawień